dotcom-rendering/src/components/marketing/gutters/GutterAsk.tsx

```diff
@@ -0,0 +1,221 @@
+import { css, ThemeProvider } from '@emotion/react';
+import type { OphanAction } from '@guardian/libs';
+import { palette, space, textSans15 } from '@guardian/source/foundations';
+import {
+	LinkButton,
+	SvgArrowRightStraight,
+} from '@guardian/source/react-components';
+import type {
+	GutterContent,
+	GutterProps,
+} from '@guardian/support-dotcom-components/dist/shared/types/props/gutter';
+import { useCallback, useEffect } from 'react';
+import { useIsInView } from '../../../lib/useIsInView';
+import type { ReactComponent } from '../lib/ReactComponent';
+import {
+	addRegionIdAndTrackingParamsToSupportUrl,
+	createClickEventFromTracking,
+} from '../lib/tracking';
+
+export const GutterAskWrapper: ReactComponent<GutterProps> = (
+	props: GutterProps,
+) => {
+	const { content, tracking, submitComponentEvent } = props;
+	const { abTestName, abTestVariant, componentType, campaignCode } = tracking;
+	const { baseUrl } = props.content.cta!; // TODO: cta forced to be defined - correct?
```

Comment:
should the cta be a required field?

Reply:
Agreed that it can be undefined and, if so we hide the CTA.